src/k8s-engine/wsl.ts

```diff
@@ -512,6 +527,55 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
     }
   }
 
+  /**
+   * Write out /etc/hosts in the main distribution, copying the bulk of the
+   * contents from the data distribution.
+   */
+  protected async writeHostsFile() {
+    await this.progressTracker.action('Updating /etc/hosts', 50, async() => {
+      const contents = await fs.promises.readFile(`\\\\wsl$\\${ DATA_INSTANCE_NAME }\\etc\\hosts`);
+      const hosts = ['host.rancher-desktop.internal', 'host.docker.internal'];
```

Comment:
No, because Windows never uses lima. We should be using `host.rancher-desktop.internal` in both places (and use the lima name for our internal lima-specific implementation).

Reply:
ah, right